src/qt/forms/modaloverlay.ui

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <ui version="4.0">
  <class>ModalOverlay</class>
- <widget class="ModalOverlay" name="ModalOverlay">
+ <widget class="QWidget" name="ModalOverlay">
```

Comment:
Right, a customwidget entry for the parent widget is always unnecessary, as it's not instantiated by the generated code.